Cargo.toml

```diff
@@ -398,7 +398,7 @@ wicket-common = { path = "wicket-common" }
 wicketd-client = { path = "clients/wicketd-client" }
 zeroize = { version = "1.7.0", features = ["zeroize_derive", "std"] }
 zip = { version = "0.6.6", default-features = false, features = ["deflate","bzip2"] }
-zone = { version = "0.3", default-features = false, features = ["async"] }
+zone = { version = "0.3", default-features = false, features = ["async", "sync"] }
```

Comment:
`Zones::list_blocking` is called a couple layers down the stack from `DumpSetupWorker::archive_files`, which is called within a MutexGuard - I was hoping to avoid questions of cancellation-safety issues this way.